alerts/plugins/ip_source_enrichment.json.conf

```diff
@@ -0,0 +1,14 @@
+{
+  "known": [
+    {
+      "ipVersion": 4,
+      "range": "8.32.0.0/16",
+      "format": "{0} is in OFFICE1."
```

Comment:
Instead of having a single human-readable string containing the data item, it may be better to have a data item for `location`. Easier to parse if needed.

So:
```
      "ipVersion": 4,
      "range": "8.32.0.0/16",
      "location:" "OFFICE1",
      "format": "{1} is in {2}."
```